packages/fluentui/docs/src/examples/components/Datepicker/BestPractices/DatepickerBestPractices.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+import ComponentBestPractices from '../../../../components/ComponentBestPractices';
+
+const doList = [
+  'Use `Form.Datepicker` example for cases when you need to announce validation/requirement errors to the user.',
+  'Add `aria-*` attributes to describe advanced keyboard navigation shortcuts.',
+];
+
+const dontList = ['Use standalone `Datepicker` with allowed user manual date input.'];
```

Comment:
@pompomon I don't exactly understand the reasons here :( What is the back story??